### PR TITLE
Rename "Media Player" story to "MediaPlayer"

### PR DESCRIPTION
**Pull request by hotinglok**
**Overall change:** 
As the title says. Done in order to follow convention of all other components.

**Code changes:**
- Renamed `Media Player` in storybook to `MediaPlayer`.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 2.1.1 | [PR#2424](https://github.com/bbc/psammead/pull/2532) Rename "Media Player" story to "MediaPlayer"
```
Comment (OlgaLyubin) [inline packages/components/psammead-media-player/CHANGELOG.md]:
```suggestion
| 2.1.1 | [PR#2532](https://github.com/bbc/psammead/pull/2532) Rename "Media Player" story to "MediaPlayer"
```

Comment (hotinglok) [reply packages/components/psammead-media-player/CHANGELOG.md]: this never happened. Totally.

**Review by OlgaLyubin — APPROVED** [review]
LGTM

**Review by DenisHdz — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Review by FK78 — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Pull request merged**